docs/data-masking-component-functions.md

```diff
@@ -0,0 +1,385 @@
+1 dialing code and is in the 555 area code. The 555 area code
+is not valid for any U.S. phone number.</td>
+            </tr>
+    <tr>
+        <td>gen_rnd_ssn()</td>
+        <td>Generates a random, non-legitimate US Social Security Number in an AAA-BBB-CCCC format. This function should only be used for test purposes.</td>
+           </tr>
+</table>
+
+#### Examples
+
+An example of `gen_range(lower, upper)`:
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT gen_range(10, 100);
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +--------------------------------------+
+    | gen_range(10,100)                    |
+    +--------------------------------------+
+    | 56                                   |
+    +--------------------------------------+
+    ```
+
+An example of `gen_range(lower, upper)` with negative numbers:
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT gen_range(-100,-80);
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +--------------------------------------+
+    | gen_range(-100,-80)                  |
+    +--------------------------------------+
+    | -91                                  |
+    +--------------------------------------+
+    ```
+
+An example of `gen_rnd_email()`:
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT gen_rnd_email();
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +---------------------------------------+
+    | gen_rnd_email()                       |
+    +---------------------------------------+
+    | sma.jrts@example.com                  |
+    +---------------------------------------+
+    ```
+
+An example of `mask_pan(gen_rnd_pan())`:
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT mask_pan(gen_rnd_pan());
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +-------------------------------------+
+    | mask_pan(gen_rnd_pan())             |
+    +-------------------------------------+
+    | XXXXXXXXXXXX4444                    |
+    +-------------------------------------+
+    ```
+
+An example of `gen_rnd_us_phone()`:
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT gen_rnd_us_phone();
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +-------------------------------+
+    | gen_rnd_us_phone()            |
+    +-------------------------------+
+    | 1-555-635-5709                |
+    +-------------------------------+
+    ```
+
+An example of `gen_rnd_ssn()`:
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT gen_rnd_ssn()
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +-----------------------------+
+    | gen_rnd_ssn()               |
+    +-----------------------------+
+    | 995-33-5656                 |
+    +-----------------------------+
+    ```
+
+
+
+Use a selected dictionary to generate random terms. The dictionary must be loaded from a file with the following characteristics:
```

Comment:
This is definitely not about `gen_rnd_ssn`, but something else.

Also, in the component, nothing is loaded from a file, that was the plugin.

Reply:
Removed.